pkg/kubectl/cmd/util/factory_test.go

```diff
@@ -37,6 +37,7 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
@xiangpengzhao 
I checked "Allow edits from maintainers."
May some maintainers can do these operation instead of me.